Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -221,6 +221,70 @@ INT_VECT EmbedMultipleConfs(
     bool enforceChirality = true, bool useExpTorsionAnglePrefs = false,
     bool useBasicKnowledge = false, bool verbose = false,
     double basinThresh = 5.0);
+
+struct EmbedParameters {
+  unsigned int maxIterations;
+  int numThreads;
+  int randomSeed;
+  bool clearConfs;
+  bool useRandomCoords;
+  double boxSizeMult;
+  bool randNegEig;
+  unsigned int numZeroFail;
+  const std::map<int, RDGeom::Point3D> *coordMap;
+  double optimizerForceTol;
+  bool ignoreSmoothingFailures;
+  bool enforceChirality;
+  bool useExpTorsionAnglePrefs;
+  bool useBasicKnowledge;
+  bool verbose;
+  double basinThresh;
+  double pruneRmsThresh;
+  EmbedParameters()
+      : maxIterations(0),
+        numThreads(1),
```

Comment:
Do we have a sentinel (i.e. -1) for use all the threads?  I forget...  Might be useful to have a constant somewhere if so.  i.e. 

```
embedParams.numThreads = ALL_CORES;
```